w3w-autosuggest/public/class-w3w-autosuggest-public.php

```diff
@@ -107,17 +107,18 @@ public function enqueue_scripts() {
     global $woocommerce;
 
     $settings = get_option( W3W_SETTINGS_NAME );
+    $has_woocommerce = class_exists( 'woocommerce' );
     $exposed_settings['version'] = $settings['version'];
     $exposed_settings['php_version'] = phpversion();
     $exposed_settings['wp_version'] = $wp_version;
     $exposed_settings['wc_version'] = isset( $woocommerce ) ? $woocommerce->version : 'N/A';
     $exposed_settings['api_key'] = $settings['api_key'];
     // Determine if WordPress is running with WooCommerce enabled
     // https://woocommerce.com/document/query-whether-woocommerce-is-activated/
```

Comment:
This comment should also be moved up to where `class_exists( 'woocommerce' );` is used

Reply:
Or get rid of it altogether perhaps, it's only really helpful as it links to the WooCommerce site